deploy/gke-marketplace-app/benchmark/model-store/bert_base_tf_cpu/config.pbtxt

```diff
@@ -1,3 +1,29 @@
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Need to use "NVIDIA CORPORATION & AFFILIATES" now for all new and updated copyrights

Reply:
Got it, I updated every file in the app. @deadeyegoodwin